### ISSUE TITLE
Shape dimensions should only be int or float

### ISSUE THREAD

**Issue by yashu-seth** [pydy/pydy#299]
```python
>>> from pydy.viz.shapes import Sphere,Cube
>>> c = Cube('abcd')
>>> s = Sphere('xyz')
>>> print(s)
Sphere unnamed color:grey material:default radius:xyz
>>> print(c)
Cube unnamed color:grey material:default length:abcd
```

**Comment by yashu-seth** [comment]
@moorepants I don't think this is the expected beaviour. If you also feel the same, I can send a PR.

**Comment by moorepants** [comment]
We work somewhat on the "garbage in/garbage out" principal instead of having tons of type checks, but if you'd like to submit a PR with the type checks I will accept it. Thanks.

**Comment by yashu-seth** [comment]
@moorepants okay i think i'll pass. Can you please get me started with some beginner friendly issues?

**Comment by moorepants** [comment]
This would be a good one to start with: https://github.com/pydy/pydy/issues/296

You have to run the examples and see which ones are using deprecated code.

**Comment by yashu-seth** [comment]
@moorepants thank you. I'll look into it.

**Comment by oliverlee** [comment]
I think the existing PR fixes this but I can't check for a while.

**Comment by yashu-seth** [comment]
@oliverlee yes, I think you have already sent a PR for this.

**Comment by oliverlee** [comment]
@yashu-seth I just realized I won't have wifi for a while. so if you'd like to work on it, please do. You can take a look at the existing PR for a starting point.

**Comment by moorepants** [comment]
Note that the shapes support sympy expressions as dimensions, so they shouldn't "only be int or float".

I actually don't see any reason to make these specific type checks. Python code typically follows duck typing and I'm fine if we use the garbage in/garbage out philosophy here. You simply need to have clear docstrings that say what should be passed in. We don't have to have tons of type checks.
